containers/src/Kokkos_StaticCrsGraph.hpp

```diff
@@ -299,43 +299,23 @@ class StaticCrsGraph {
   using row_block_type =
       View<const size_type*, array_layout, device_type, memory_traits>;
 
-  entries_type entries;
-  row_map_type row_map;
-  row_block_type row_block_offsets;
+  entries_type entries{};
+  row_map_type row_map{};
+  row_block_type row_block_offsets{};
```

Comment:
Just noting that these in-class default member initializers were not merged into Kokkos Kernels.

Reply:
Yes, I just need to gather some evidence first basically...

Reply:
My first guess would be that as long as this PR is not merged, and if some configuration is compiled using the deprecated Kokkos 4 code, they will be using Kokkos kernels with the "old" version of this class, *i.e.* without the copy-convert constructor. And so it will break in the CRS matrix for instance (see the changes in https://github.com/kokkos/kokkos-kernels/pull/2654).

Reply:
@cwpearson has submitted a couple of PRs to Kokkos Kernels to resolve our problems.
@romintomasetti unfortunately our nightly builds are only accessible internally...

Reply:
I can remove the in-class default member initializers to align to what's been merged in `kokkos-kernels`.